From 1c3d615d93b20d10c2729478d5104977dd9af23f Mon Sep 17 00:00:00 2001
From: Angelo Compagnucci <angelo.compagnucci@gmail.com>
Date: Wed, 5 Aug 2015 12:59:45 +0200
Subject: [PATCH] Fixing initialization of have_vasprintf

This patch initialize properly have_vasprintf in case vasprint function is found.
Solves multiple definition of `vasprintf' error in case vasprint is not properly detected.

Patch is upstream:

   https://github.com/mono/mono/commit/40c171799b671718969ee28a02f92884d7fd181e

Signed-off-by: Angelo Compagnucci <angelo.compagnucci@gmail.com>
---
 eglib/configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eglib/configure.ac b/eglib/configure.ac
index 5281419..4bf91e5 100644
--- a/eglib/configure.ac
+++ b/eglib/configure.ac
@@ -135,6 +135,7 @@ AC_CHECK_SIZEOF(void *)
 AC_CHECK_SIZEOF(long)
 AC_CHECK_SIZEOF(long long)
 AC_CHECK_FUNCS(strlcpy stpcpy strtok_r rewinddir vasprintf)
+AC_CHECK_FUNC(vasprintf, have_vasprintf=yes)
 AC_CHECK_FUNCS(getrlimit)
 
 #
